### PR TITLE
chore: buffalo camemberts fix

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -39892,42 +39892,58 @@ ciqual_food_code:en:12001
 ciqual_food_name:en:Camembert cheese, from cow's milk
 
 <en:Camemberts
+en:Buffalo camemberts
+fr:Camemberts au lait de bufflonne
+
+<en:Buffalo camemberts
+<en:Pasteurized cheeses
+en:Camemberts from pasteurized bufalo milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39900]:
```suggestion
en:Camemberts from pasteurized buffalo milk
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -39892,42 +39892,58 @@ ciqual_food_code:en:12001
...
+fr:Camemberts au lait pasteurisé de bufflonne
+
+<en:Buffalo camemberts
 <en:Unpasteurised cheeses
-en:Camemberts from raw milk, Camembert made with raw milk, Camembert cheese from cow's milk from raw milk
+en:Camemberts from raw bufalo milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39905]:
```suggestion
en:Camemberts from raw buffalo milk
```

Comment (moon-rabbitOFF) [reply taxonomies/categories.txt:39905]: I changed it, thank you

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**